cmd/agent/app/integrations_nix.go

```diff
@@ -11,36 +11,20 @@ package app
 import (
 	"os"
 	"path/filepath"
-
-	"github.com/DataDog/datadog-agent/pkg/util/executable"
 )
 
 const (
-	pythonBin = "python"
+	pythonBin     = "python"
+	downloaderBin = "datadog-checks-downloader"
```

Comment:
This doesn't seem to be used anywhere? 🤔

Reply:
indeed, forgot to remove that